nextvibe/configs/settings/base.py

```diff
@@ -0,0 +1,178 @@
+"""
+Django settings for nextvibe project.
+
+Generated by 'django-admin startproject' using Django 5.2.6.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/5.2/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/5.2/ref/settings/
+"""
+import os
+from pathlib import Path
+import environ
+from datetime import timedelta
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+# env 로더
+env = environ.Env(
+    DEBUG = (bool, False),
+    SECRET_KEY=(str, "dev-secret"),
+)
+environ.Env.read_env(os.path.join(BASE_DIR, '.env'))
```

Comment:
❗ env_example 폴더와 다른 구조로 env 폴더를 구성하신 것 같아요~
```suggestion
environ.Env.read_env(os.path.join(BASE_DIR, 'env', '.env.base'))
```